libs/core/src/lib/utils/decorators/apply-css-class.decorator.ts

```diff
@@ -0,0 +1,14 @@
+import { ELEMENT_REF_EXCEPTION } from '../public_api';
+
+export function applyCssClass(target: any, propertyKey: string, descriptor: PropertyDescriptor) {
```

Comment:
I know that in general this function is descriptive, but could you describe it with some comments `/** */`. Also let's describe all of these arguments.

Reply:
done